client/src/components/Message/MessageList.tsx

```diff
@@ -1,18 +1,32 @@
 import React from "react";
+import styled from "styled-components";
 import { Member, Message as IMessage } from "graphql/generated/graphql";
 
 import Message from "components/Message/Message";
 import { useAuthContext } from "contexts/AuthContext";
 
 interface IMessageList {
   messages?: IMessage[];
+  onScroll?: (event: any) => void;
 }
 
-const MessageList: React.FC<IMessageList> = ({ messages }) => {
+const StyledMessageList = styled.section`
+  overflow-y: scroll;
+  height: calc(100vh - 150px);
+`;
+
+const MessageList = React.forwardRef<
+  HTMLElement,
+  IMessageList
+>(({ messages, onScroll }, ref) => {
   const { user } = useAuthContext();
 
   return (
-    <div>
+    <StyledMessageList
+      ref={ref}
+      onScrollCapture={onScroll}
```

Comment:
We are gonna scroll capture in this container instead of the whole Dashboard Body